osu.Game/Screens/Edit/LegacyEditorBeatmapPatcher.cs

```diff
@@ -114,11 +114,11 @@ public PassThroughWorkingBeatmap(IBeatmap beatmap)
 
             protected override IBeatmap GetBeatmap() => beatmap;
 
-            protected override Texture GetBackground() => throw new NotImplementedException();
+            protected override Texture GetBackground() => Background;
 
-            protected override Track GetBeatmapTrack() => throw new NotImplementedException();
+            protected override Track GetBeatmapTrack() => Track;
 
-            protected internal override ISkin GetSkin() => throw new NotImplementedException();
+            protected internal override ISkin GetSkin() => Skin;
```

Comment:
these shouldn't be implemented. rationale as above, nobody is expecting a beatmap patcher to be providing tracks or skins.